packages/@coorpacademy-player-services/src/test/fixtures/recommendations.js

```diff
@@ -1,17 +1,35 @@
+// @flow strict
+
 import reduce from 'lodash/fp/reduce';
 import _find from 'lodash/fp/find';
 
+import type {Level, Recommendation} from '../../types';
 import levelsData from './data/levels';
 import recommendations from './data/recommendations';
 
-const levels = reduce((map, object) => map.set(object.ref, object), new Map(), levelsData);
+const levels: Map<string, Level> = reduce(
+  (map, object) => map.set(object.ref, object),
+  new Map(),
+  levelsData
+);
 
-const find = (type, ref) => {
+const find = (type: string, ref: string): Promise<Array<Recommendation>> => {
   return Promise.resolve(recommendations);
 };
 
-const getNextLevel = ref => {
-  const {name, level} = levels.get(ref);
+const getNextLevel = (ref: string): Level | void => {
+  const _level: Level | void = levels.get(ref);
```

Comment:
It's not useful to type this one, the function has return type

Reply:
✅